src/components/Hero/ContentHero/index.tsx

```diff
@@ -1,58 +1,54 @@
 import { Box, Heading, HStack, SimpleGrid, Stack, Text } from "@chakra-ui/react"
 
-import type { CommonHeroProps } from "@/lib/types"
+import { CommonHeroProps } from "@/lib/types"
 
-import Breadcrumbs, { type BreadcrumbsProps } from "@/components/Breadcrumbs"
-import { CallToAction } from "@/components/Hero/CallToAction"
+import Breadcrumbs, { BreadcrumbsProps } from "@/components/Breadcrumbs"
 import { Image } from "@/components/Image"
 
+import { CallToAction } from "../CallToAction"
+
 export interface ContentHeroProps extends Omit<CommonHeroProps, "header"> {
   breadcrumbs: BreadcrumbsProps
 }
 
-const ContentHero = ({
-  breadcrumbs,
-  heroImgSrc,
-  buttons,
-  title,
-  description,
-}: ContentHeroProps) => (
-  <Box bgImg="bgMainGradient">
-    <SimpleGrid columns={{ base: 1, lg: 2 }} maxW="1536px" mx="auto" gap="4">
-      <Box
-        height={{ base: "300px", md: "400px", lg: "full" }}
-        order={{ lg: 1 }}
-      >
-        <Image
-          alt=""
-          src={heroImgSrc}
-          priority
-          style={{ objectFit: "contain" }}
-          boxSize="full"
-        />
-      </Box>
-      <Stack p={{ base: "8", lg: "16" }} spacing="9" justify="center">
-        <Breadcrumbs {...breadcrumbs} />
-        <Stack spacing="6">
-          <Heading as="h1" size="2xl">
-            {title}
-          </Heading>
-          <Text fontSize="lg">{description}</Text>
-          {buttons && (
+const ContentHero = (props: ContentHeroProps) => {
+  const { breadcrumbs, heroImg, buttons, title, description } = props
+  return (
+    <Box bgImg="bgMainGradient">
+      <SimpleGrid columns={{ base: 1, lg: 2 }} maxW="1536px" mx="auto" gap="4">
+        <Box
+          height={{ base: "300px", md: "400px", lg: "full" }}
+          order={{ lg: 1 }}
+        >
+          <Image
+            src={heroImg}
+            alt=""
+            loading="eager"
+            objectFit="contain"
```

Comment:
We need to keep the `priority` flag in the hero image to preload the image (also to be consistent with the other hero images). And the `objectFit` prop needs to be passed inside the `style` prop in order to make it work with the Next Image component.

```suggestion
            priority
            style={{ objectFit: "contain" }}
```